### PR TITLE
Hello world translation

### PR DESCRIPTION
**Pull request by alejandronanez**
Related to #4

### CODE REVIEW

File: content/docs/hello-world.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ prev: cdn-links.html
 next: introducing-jsx.html
 ---
 
-The smallest React example looks like this:
+El ejemplo más pequeño de React se ve así:
```
Comment (dmoralesm) [inline content/docs/hello-world.md]:
```suggestion
El más pequeño de los ejemplos de React se ve así:
```
How it sounds to you?

Comment (alejandronanez) [reply content/docs/hello-world.md]: Sounds good @dmoralesm, thanks for the suggestion.

File: content/docs/hello-world.md [hunk 2]
```diff
@@ -15,36 +15,34 @@ ReactDOM.render(
...
 
->Note
+>Nota
 >
->This guide occasionally uses some of the newer JavaScript syntax in the examples. If you haven't worked with JavaScript in the last few years, [these three points](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) should get you most of the way.
-
+> Esta guía ocasionalmente usa la nueva syntaxis de JavaScript en los ejemplos. Si no has trabajado con JavaScript en los últimos años, [estos tres puntos](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) son un buen resumen.
```
[the remark this answers is not in the document]
Comment (carburo) [reply content/docs/hello-world.md]: I actually like it. :+1:

Comment (carburo) [inline content/docs/hello-world.md]: I hate being the typo guy. :smiley: 
```suggestion
> Esta guía ocasionalmente usa la nueva sintaxis de JavaScript en los ejemplos. Si no has trabajado con JavaScript en los últimos años, [estos tres puntos](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) son un buen resumen.
```

File: content/docs/hello-world.md [hunk 3]
```diff
@@ -15,36 +15,34 @@ ReactDOM.render(
...
 
-## Let's Get Started!
+## Empecemos!
 
-Keep scrolling down, and you'll find the link to the [next chapter of this guide](/docs/introducing-jsx.html) right before the website footer.
+Sigue bajando, y encontraras el link al [siguiente capítulo de esta guía](/docs/introducing-jsx.html) justo antes del pie de la página.
```
Comment (carburo) [inline content/docs/hello-world.md]:
```suggestion
Sigue bajando, y encontrarás el link al [siguiente capítulo de esta guía](/docs/introducing-jsx.html) justo antes del pie de la página.
```

File: content/docs/hello-world.md [hunk 4]
```diff
@@ -15,36 +15,34 @@ ReactDOM.render(
...
+React es una biblioteca Javascript, y vamos a asumir que tienes un entendimiento básico del lenguaje Javascript. **Si no te sientes muy seguro, te recomendamos [hacer un tutorial de Javascript](https://developer.mozilla.org/es/docs/Web/JavaScript/Una_re-introducci%C3%B3n_a_JavaScript) para comprobar tu nivel de conocimiento** y permitirte seguir adelante con esta guía sin perderte. Te debería tomar entre 30 minutos y una hora, and so we'll assume you have a basic understanding of the JavaScript language. **If you don't feel very confident, we recommend [going through a JavaScript tutorial](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) to check your knowledge level** and enable you to follow along this guide without getting lost. It might take you between 30 minutes and an hour, pero como resultado, no tendrás que sentir que estás aprendiendo React y JavaScript al mismo tiempo.
```
Comment (carburo) [inline content/docs/hello-world.md]:
```suggestion
React es una biblioteca Javascript, y vamos a asumir que tienes un entendimiento básico del lenguaje Javascript. **Si no te sientes muy seguro, te recomendamos [hacer un tutorial de Javascript](https://developer.mozilla.org/es/docs/Web/JavaScript/Una_re-introducci%C3%B3n_a_JavaScript) para comprobar tu nivel de conocimiento** y permitirte seguir adelante con esta guía sin perderte. Te debería tomar entre 30 minutos y una hora, pero como resultado, no tendrás que sentir que estás aprendiendo React y JavaScript al mismo tiempo.
```

File: content/docs/hello-world.md [hunk 5]
```diff
@@ -15,36 +15,34 @@ ReactDOM.render(
...
+Este es el primer capítulo en una guía paso a paso sobre los principales conceptos de React. Puedes encontrar una lista de todos los capítulos en la barra de navigación lateral. Si estás leyendo esto desde un dispositivo móvil, puedes acceder a la navegación presionando el botón en la esquina inferior derecha de tu pantalla.
 
-Every chapter in this guide builds on the knowledge introduced in earlier chapters. **You can learn most of React by reading the “Main Concepts” guide chapters in the order they appear in the sidebar.** For example, [“Introducing JSX”](/docs/introducing-jsx.html) is the next chapter after this one.
+Cada capítulo en esta guía se construye en base al conocimiento presentado en capítulos anteriores. **Tu puedes aprender la mayoría de React leyendo la guía de conceptos "Conceptos Principales" en el orden que aparecen en la barra lateral.** Por ejemplo, ["Introducción a JSX”](/docs/introducing-jsx.html) es el siguiente capítulo después de este.
```
Comment (carburo) [inline content/docs/hello-world.md]: I would omit "tú", but that's just me.
```suggestion
Cada capítulo en esta guía se construye en base al conocimiento presentado en capítulos anteriores. **Tú puedes aprender la mayoría de React leyendo la guía de conceptos "Conceptos Principales" en el orden que aparecen en la barra lateral.** Por ejemplo, ["Introducción a JSX”](/docs/introducing-jsx.html) es el siguiente capítulo después de este.
```

File: content/docs/hello-world.md [hunk 6]
```diff
@@ -15,36 +15,34 @@ ReactDOM.render(
...
 
-## How to Read This Guide
-
-In this guide, we will examine the building blocks of React apps: elements and components. Once you master them, you can create complex apps from small reusable pieces.
+En esta guía, examinaremos los componentes básicos de las aplicaciones React: elementos y componentes. Una vez los domines, podrás crear aplicaciones complejas a partir de pequeñas piezas reutilizables. 
 
 >Tip
 >
->This guide is designed for people who prefer **learning concepts step by step**. If you prefer to learn by doing, check out our [practical tutorial](/tutorial/tutorial.html). You might find this guide and the tutorial complementary to each other.
+>Esta guía está diseñada para personas que prefieren **aprender los conceptos paso a paso**. Si tu prefieres aprender haciendo, revisa nuestro [tutorial práctico](/tutorial/tutorial.html). Encontrarás que esta guía y el tutorial se complementan el uno al otro.
```
Comment (carburo) [inline content/docs/hello-world.md]:
```suggestion
>Esta guía está diseñada para personas que prefieren **aprender los conceptos paso a paso**. Si tú prefieres aprender haciendo, revisa nuestro [tutorial práctico](/tutorial/tutorial.html). Encontrarás que esta guía y el tutorial se complementan el uno al otro.
```

File: content/docs/hello-world.md (hunk 2 above)
Comment (alejandronanez) [reply content/docs/hello-world.md]: 😆

**Pull request merged**